plugins/gozero/rest/config.go

```diff
@@ -0,0 +1,9 @@
+package rest
+
+//skywalking:config gozero
+var config struct {
+	CollectRequestParameters bool   `config:"collect_request_parameters"` // CollectRequestParameters is used to determine whether to collect request parameters.
+	CollectResponseData      bool   `config:"collect_response_data"`      // CollectResponseData is used to determine whether to collect response data.
+	DataSensitiveFields      string `config:"data_sensitive_fields"`      // DataSensitiveFields is used to determine the sensitive fields in the request and response data.
```

Comment:
I can't see any place have use this configuration.

Reply:
Alright, I have now retained only the necessary configurations and will wait to further develop and refine it later.